File: examples/canonical-azure-functions/blob-trigger-receipt-ocr/python/tests/integration/lane-b-ephemeral-real/test_handler.py
from examples.canonical_azure_functions_import import load_python_handler

handler = load_python_handler(
    "examples/canonical-azure-functions/blob-trigger-receipt-ocr/python/src/handler.py"
)
assert handler.process_blob({"name": "lane-b/receipt.pdf", "version": "real-1"})["event_grid_event"]
print("lane-b Azure blob receipt probe passed")

File: examples/canonical-azure-functions/blob-trigger-receipt-ocr/typescript/tests/integration/lane-b-ephemeral-real/test_handler.js
const assert = require("node:assert/strict");
const { processBlob } = require("../../../src/handler.js");

const result = processBlob({ name: "lane-b/receipt.pdf", version: "real-cloud-etag", total: 21 });
assert.equal(result.event_grid_event.data.receipt_id, result.idempotency_key);
console.log("lane-b Azure blob receipt ocr typescript probe passed");
